crates/topos-tce-broadcast/src/constant.rs

```diff
@@ -7,6 +7,24 @@ lazy_static! {
             .ok()
             .and_then(|s| s.parse().ok())
             .unwrap_or(2048);
+    /// Size of the channel between double echo and the task manager
+    pub static ref TASK_MANAGER_CHANNEL_SIZE: usize =
+        std::env::var("TOPOS_TASK_MANAGER_CHANNEL_SIZE")
```

Comment:
we could have been prefixing the variable by the component name like `TOPOS_BROADCAST_TASK_MANAGER_CHANNEL_SIZE` in order to avoid any overlap in the future